package.json

```diff
@@ -61,11 +68,11 @@
     "module-alias": "^2.2.2",
     "nodemon": "^2.0.15",
     "preact": "^10.5.15",
+    "prettier": "^2.7.1",
     "tinyduration": "^3.2.2",
     "tld-extract": "^2.0.1",
-    "ts-node": "^10.4.0",
```

Comment:
removing `ts-node` from dependencies is probably a mistake because it is used to start the demo

Reply:
Updated package.json